pages/gallery/_.vue

```diff
@@ -54,22 +74,30 @@
           return {
             description: response.items[0].fields.description,
             images: response.items[0].fields.hasPart,
-            title: response.items[0].fields.name
+            title: response.items[0].fields.name,
+            canonicalURL: null
```

Comment:
The naming of this is inconsistent with that of `shareMediaUrl`. Rename it to `canonicalUrl`.